### PR TITLE
Launch hooks

### PR DESCRIPTION
**Pull request by convolu**
This PR replaces https://github.com/KanoComputing/kano-profile/pull/338

### CODE REVIEW

**Comment by convolu** [comment]
@Ealdwulf @pazdera @tombettany Could you please review?

cc @andreiconstantinescu

**Comment by convolu** [comment]
@alex5imon I didn't forget you :-). Could you please review as well?

**Comment by convolu** [comment]
Sorry for sneaking in the gitignore file with this commit, it had to happen sometime :-)

**Comment by pazdera** [comment]
Can't see anything wrong :shipit:

**Comment by alex5imon** [comment]
@convolu you might want to bump the version of this package

**Comment by convolu** [comment]
Bumped to `2.2.0-4` as per your comment. Could I merge?

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,20 @@
 *.pyc
 /kano
-**.DS_Store
\ No newline at end of file
+**.DS_Store
+*.swp
```
Comment (tombettany) [inline .gitignore]: This is a welcome addition.

File: kano_world/hooks/launch.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python
```
Comment (tombettany) [inline kano_world/hooks/launch.py]: Isn't this file imported? If so, does it need the shebang?

**Comment by tombettany** [comment]
Other than the minor comment above, looks fine to me.

File: kano_world/hooks/launch.py (hunk 2 above)
Comment (convolu) [inline kano_world/hooks/launch.py]: Indeed it doesn't. Blame my too broad copy-paste skills, I will remove it from the other file as well

**Comment by tombettany** [comment]
@convolu I hope you don't mind but I rebased on the new `master` to permit merging. The merge conflict was due to both this and `master` bumping the package version. My modification to your commit can be found [here](https://github.com/KanoComputing/kano-profile/commit/efd81bfad73a3d5340f26910588ff5dbf8eb5d4d)

**Comment by convolu** [comment]
@tombettany Made changes as per your comments

**Pull request merged**